go/test/endtoend/vreplication/vstream_test.go

```diff
@@ -1127,7 +1127,7 @@ func TestVStreamPushdownFilters(t *testing.T) {
 	require.NoError(t, err)
 
 	// Coordinate go-routines.
-	streamCtx, streamCancel := context.WithTimeout(context.Background(), 1*time.Minute)
+	streamCtx, streamCancel := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
@mattlord This test was taking around 90 seconds to complete. Can you please let me know if there's a specific reason the timeout was set to 1 min? Is it okay to decrease it? Setting it to 10 sec still gets around 2 copy phase rows and ~70 running phase rows.

Reply:
This is an end to end test. 90 seconds is quite fast for an end to end test. At the same time... cutting it in half would be OK.